docs/prms/oi1.rst

```diff
@@ -17,4 +17,10 @@ OI1 takes some optional arguments:
 * mu: controls the degree-based negative prizes (default 0.0)
 * noise: Standard Deviation of the gaussian noise added to edges in Noisy Edges Randomizations
 * g: (Gamma) multiplicative edge penalty from degree of endpoints
```

Comment:
I noticed this is the OI2 definition, not the OI1 definition. We should change it to:
`
```suggestion
* g: msgsteiner reinforcement parameter that affects the convergence of the solution and runtime, with larger values leading to faster convergence but suboptimal results (default 0.001)
```